1.MLP_Binary_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,627 @@
+#!/bin/env/python
+from datetime import date
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+import seaborn as sns
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from sklearn.metrics import (
+    accuracy_score,
+    auc,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    precision_score,
+    recall_score,
+    roc_auc_score,
+    roc_curve,
+)
+from sklearn.model_selection import train_test_split
+
+from .parameters import Parameters
+
+params = Parameters()
+
+
+def data_split(X_vals, y_vals, train, val, test):
+    """
```

Comment:
I really like this documentation, well done! 

One important question: What are the function arguments actually? Consider adding python type hints and also please use our standard documentation syntax (I believe it is called sphinx numpy). See @axiomcura's example here: https://github.com/WayScience/CytoSnake/blob/main/cytosnake/common/errors.py#L93

Please consider making these changes to all functions. It will help a bunch in the long run!! :D

Reply:
Finished, I agree that this will help in the long run :D